changelogs/unreleased/gh-tarantool-10142-box-ctl-wal-sync-lua.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua
+
+* Added `box.ctl.wal_sync()` function which uses C `wal_sync()` and wait until
+  submitted writes are flushed to the disk (gh-10142).
```

Comment:
```suggestion
* Added `box.ctl.wal_sync()` function which uses C `wal_sync()` and waits until
  submitted writes are flushed to the disk (gh-10142).
```

Reply:
Thanks! I fxed that but then I actually rewrote most of that changelog

Reply:
@lenkis, please, chech the changelog one more time. It was significantly changed